bodhi/server/migrations/versions/68f9ccb1f388_add_a_syncing_repo_composestate_enum_.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) YEAR AUTHOR
```

Comment:
Let's fill in the year and author:

```
# Copyright (c) 2018 Mattia Verga
```

Reply:
This is a minor enough of a change that I'll just force push your branch and fix it for you. Fix incoming!